store/contentSchemas.json

```
@@ -0,0 +1,103 @@
+{
+  "definitions": {
+    "Shelf": {
+      "description": "admin/editor.shelf.description",
+      "type": "object",
+      "properties": {
+        "category": {
+          "title": "admin/editor.productSummaryList.category.title",
+          "description": "admin/editor.productSummaryList.category.description",
+          "type": "string",
+          "isLayout": false
```

This flag is not needed
```suggestion
          "type": "string"
```